cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -3240,6 +3240,18 @@
     </description>
   </property>
 
+  <property>
+    <name>preview.runner.artifact.localizer.enabled</name>
+    <value>false</value>
+    <description>
+      Whether to enable artifact localizer sidecar container to fetch and cache artifacts.
+      This replaces the default direct access of artifact jars by preview runners.
```

Comment:
nit:
if true, artifacts are accessed through side-car container instead of directly accessing them through GCS.

Reply:
Done